statsbombpy/sb.py

```diff
@@ -121,3 +121,52 @@ def competition_events(
             [*competition_events.values()], axis=0, ignore_index=True, sort=True
         )
     return competition_events
+
+
+def frames(
+    match_id: int,
+    fmt: str = "dataframe",
+    creds: dict = DEFAULT_CREDS,
+) -> Union[pd.DataFrame, dict]:
+    if api_client.has_auth(creds) is True:
+        frames = api_client.frames(match_id, creds=creds)
+    else:
+        frames = public.frames(match_id)
+    if fmt == "dataframe":
+        frames = pd.json_normalize(
+            frames, "freeze_frame", ["event_uuid", "visible_area"]
+        )
+        frames = frames.rename(columns={"event_uuid": "id"})
```

Comment:
Are the `id` fields consistent across the frames and events i.e. they are both `int` or `str`, so can be joined on `id`?

Reply:
Yep